### PR TITLE
SSE-2583 Add permission and TTL restrictions for session storage

### PR DESCRIPTION
**Pull request by rafal-surowiec-co**

### CODE REVIEW

File: infrastructure/ci/dev-platform/frontend-template.yml [hunk 1]
```diff
@@ -113,7 +113,30 @@ Resources:
             Condition:
               ArnLike:
                 aws:SourceArn: !Sub arn:aws:ecs:${AWS::Region}:${AWS::AccountId}:*
-      ManagedPolicyArns: [ arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role ]
```
Comment (JohnWattsGDS) [inline infrastructure/ci/dev-platform/frontend-template.yml:116]: Do we not still need this policy?  If we do, then it's an array so we can have both policies attached:

`ManagedPolicyArns: [ arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role, !GetAtt FrontendECSDynamoDBAccessPolicy.Arn ]`

Or something to that effect.

Comment (rafal-surowiec-co) [reply infrastructure/ci/dev-platform/frontend-template.yml:116]: I do not think these rights are needed for frontend application:
                "ec2:DescribeTags",
                "ecs:CreateCluster",
                "ecs:DeregisterContainerInstance",
                "ecs:DiscoverPollEndpoint",
                "ecs:Poll",
                "ecs:RegisterContainerInstance",
                "ecs:StartTelemetrySession",
                "ecs:UpdateContainerInstancesState",
                "ecs:Submit*",
                "ecr:GetAuthorizationToken",
                "ecr:BatchCheckLayerAvailability",
                "ecr:GetDownloadUrlForLayer",
                "ecr:BatchGetImage",
                "logs:CreateLogStream",
                "logs:PutLogEvents"

File: backend/dynamo-db/template.yml [hunk 2]
```diff
@@ -59,8 +59,10 @@ Resources:
       KeySchema:
         - AttributeName: id
           KeyType: HASH
-
       BillingMode: PAY_PER_REQUEST
+      TimeToLiveSpecification:
+        AttributeName: expires
+        Enabled: true
```
Comment (CharlesIC) [inline backend/dynamo-db/template.yml:65]: Need to confirm the expiration period is written to the `expires` column

Comment (rafal-surowiec-co) [reply backend/dynamo-db/template.yml:65]: As first step we are setting cookie with maxAge of one hour
```        
const DynamoDBStore = connect_dynamodb({session: session});
return session({
    store: new DynamoDBStore(options),
    secret: "keyboard cat",
    cookie: {maxAge: 1000 * 60 * 60}
});
```
And then in connect-dynamodb.js it is used to set column value
```
DynamoDBStore.prototype.getExpiresValue = function (sess) {
    var expires = typeof sess.cookie.maxAge === 'number' ? (+new Date()) + sess.cookie.maxAge : (+new Date()) + oneDayInMilliseconds;
    return Math.floor(expires / 1000);
}
```

**Review by CharlesIC — COMMENTED** [review]
The changes look fine, just need to squash all the commits above main to get rid of the merge artifacts.

Also need to add the `AdminToolSessionsTableArn` export to the `DynamoDB` template, as the moment only the table name is exported.

**Comment by rafal-surowiec-co** [comment]
I have added AdminToolSessionsTableArn to export as requested by @CharlesIC

**Review by CharlesIC — APPROVED** [review]

**Pull request merged**